content/departments/engineering/teams/core-services/managed-services/telemetry-gateway.md

```diff
@@ -0,0 +1,80 @@
+The Telemetry Gateway service is the service that ingests [telemetry v2 events](https://docs.sourcegraph.com/dev/background-information/telemetry) from all Sourcegraph instances, and is available at `telemetry-gateway.sourcegraph.com`.
+
+- As of 5.2.0, [certain flags can be configured](https://docs.sourcegraph.com/dev/background-information/telemetry#enabling-telemetry-export) to export events that have been instrumented with the new APIs to Telemetry Gateway.
+- For Sourcegraph instances that prior to 5.2.0, no events are exported.
+
+## Service images
+
+Source code for Telemetry Gateway service is in [sourcegraph/sourcegraph/cmd/telemetry-gateway](https://github.com/sourcegraph/sourcegraph/tree/main/cmd/telemetry-gateway).
+The image gets built the same way as any other Sourcegraph service, i.e. with `insiders`, the standard `main`-branch and `main-dry-run` tags.
+
+## Local development
+
+For local development, please refer to its [How to set up Telemetry Gateway locally](https://docs.sourcegraph.com/dev/how-to/telemetry_gateway).
+
+## Operations
+
+> [!NOTE]
+> To get access to most resources, you’ll need to [request infrastructure access](#infrastructure-access).
+
+Here is a list of useful quick links:
+
+- Prod instance: `telemetry-gateway.sourcegraph.com` - currently only accepts real Sourcegraph licenses.
+  - [Terraform Cloud workspaces](https://app.terraform.io/app/sourcegraph/workspaces?project=prj-9XNnACvkeM1VWteC)
+  - [Cloud Run service (metrics overview)](https://console.cloud.google.com/run/detail/us-central1/telemetry-gateway/metrics?project=telemetry-gateway-prod-acae)
+  - [Service logs](https://cloudlogging.app.goo.gl/kficDmGcZdMJHPQL9)
+  - [GCP alerts](https://console.cloud.google.com/monitoring/alerting?project=telemetry-gateway-prod-acae)
+  - [GCP errors](https://console.cloud.google.com/errors?project=telemetry-gateway-prod-acae)
+- Dev instance: `telemetry-gateway.sgdev.org` - currently only accepts `dev-private` licenses.
+  - [Terraform Cloud workspaces](https://app.terraform.io/app/sourcegraph/workspaces?project=prj-nxL7Ti7x8xp6oZTU)
+  - [Cloud Run service (metrics overview)](https://console.cloud.google.com/run/detail/us-central1/telemetry-gateway/metrics?project=telemetry-gateway-dev-0050)
+  - [Service logs](https://cloudlogging.app.goo.gl/4oVGWGz1FQKVt5vm9)
+  - [GCP alerts](https://console.cloud.google.com/monitoring/alerting?project=telemetry-gateway-dev-0050)
+  - [GCP errors](https://console.cloud.google.com/errors?project=telemetry-gateway-dev-0050)
+
+### Infrastructure access
+
+The following Entitle requests are needed to get access to Telemetry Gateway service infrastructure:
+
+- [GCP project - Prod Editor](https://app.entitle.io/request?targetType=resource&duration=21600&justification=Justification%20here&integrationId=134476cb-0bd6-4c6d-a89f-e1550988bdd7&resourceId=271c1799-6172-4099-8fe1-b186ac05aa06&roleId=da83e573-1cae-4e83-a132-3d2c6065ecbb&grantMethodId=da83e573-1cae-4e83-a132-3d2c6065ecbb)
```

Comment:
Did you test this is sufficient for accessing Cloud Run-related resources on GCP? If so, i will simplify Pings handbook page as well.

Reply:
I used the project-level equivalent: 
![image](https://github.com/sourcegraph/handbook/assets/23356519/bbc84edf-d8c7-43db-89c0-04e306801a9c)

Reply:
I'm going to merge this as-is so I can start linking it, and test the permissions again when my current set expires